indigo/settings.py

```diff
@@ -283,7 +283,7 @@
                 'bower_components/underscore/underscore-min.js',
                 'bower_components/backbone/backbone.js',
                 'bower_components/backbone.stickit/backbone.stickit.js',
-                'lib/bootstrap-4.3.1/dist/js/bootstrap.bundle.min.js',
+                'lib/bootstrap-5.0.2/dist/js/bootstrap.bundle.min.js',
```

Comment:
any reason not to use the latest 5.2.x?

Reply:
hmm, no reason at all, but I see the most recent version is 5.3.2, we can go with that..